### PR TITLE
Migrate from intltool to gettext

### PR DESCRIPTION
**Pull request by yetist**

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -25,22 +25,15 @@ EXTRA_DIST =			\
 	INSTALL			\
 	README			\
 	ChangeLog		\
+	its				\
```
Comment (raveit65) [inline Makefile.am]: Position of back-slash can be improved, or not?

**Comment by raveit65** [comment]
Comparing old and new *.pot file shows me that all translations are there.
@yetist 
Beside from the little formatting issue all looks good..

File: Makefile.am (hunk 1 above)
Comment (yetist) [reply Makefile.am]: Done.

**Comment by raveit65** [comment]
I squashed formating fix in other commit.
Why an extra commit?

**Comment by yetist** [comment]
> I squashed formating fix in other commit.
> Why an extra commit?

OK, Thanks.

**Pull request merged**